pmlc/dialect/tile/transforms/passes.td

```diff
@@ -8,6 +8,11 @@ def ComputeBounds : FunctionPass<"tile-compute-bounds"> {
   let constructor = "pmlc::dialect::tile::createComputeBoundsPass()";
 }
 
+def ExpandReshape : FunctionPass<"expand-reshape"> {
```

Comment:
prefix the pass name with the dialect it works on, in this case: `tile-expand-reshape` or maybe `tile-reshape-expand`

Reply:
done